src/gameplay/game.ts

```diff
@@ -50,32 +50,27 @@ const ALLIANCES = [
   Alliance.Spy,
 ];
 
-export const NUM_PLAYERS_ON_MISSION = [
-  [2, 3, 2, 3, 3],
-  [2, 3, 4, 3, 4],
-  [2, 3, 3, 4, 4],
-  [3, 4, 4, 5, 5],
-  [3, 4, 4, 5, 5],
-  [3, 4, 4, 5, 5],
-];
```

Comment:
Could you keep this here but instead conditionally use the sinad versions in the pickingTeam phase?

Reply:
Hmm, alternatively, if you want to have the GUI update live, you can directly mutate this array when you need to. Const arrays can still be modified.

Reply:
What about keeping two global arrays, and having a reference to one of the two arrays as a field of the Game class? updateMissionSizesForSinad() can then update that reference to point to the other array.

Reply:
Hmmm, actually, something I didn't think about was that this const object is shared across all games, so modifying it directly will cause issues. We'll need to either do what you did and move it onto the Game object itself as a member, or we'll have to make a static function that is given sinadMode and returns one or the other. 

I might slightly prefer your current solution. Let me think a bit :)